lib/trip_dispatcher.rb

```diff
@@ -27,14 +37,54 @@ def inspect
               #{passengers.count} passengers>"
     end
 
+    # method that grabs first available driver
+    def select_available_driver
+      @available_driver = @drivers.find {|driver| driver.status == :AVAILABLE}
+      return @available_driver
+    end
+
+    # Wave 3 method that allows for an in-progress trip to be made
+    def request_trip(passenger_id)
+      @passenger = self.find_passenger(passenger_id)
+     
+      # handles case when there are no available drivers
+      if @drivers.find {|driver| driver.status == :AVAILABLE}.nil?
+        raise ArgumentError.new("No available drivers.")
```

Comment:
This isn't an issue with the arguments here.  You should use a custom error class here.

For example at the top of the file you can define:
```ruby
  class NoDriverError < StandardError
  end
```

And then: 
```suggestion
        raise NoDriverError.new("No available drivers.")
```